src/components/HotspotListItem.tsx

```diff
@@ -70,79 +70,82 @@ const HotspotListItem = ({
 
   return (
     <Box marginBottom="xxs">
-      <TouchableOpacityBox
-        backgroundColor="grayBox"
-        flexDirection="row"
-        justifyContent="space-between"
-        alignItems="center"
-        padding="m"
-        onPress={handlePress}
-      >
-        <Box
-          flexDirection="row"
-          justifyContent="space-between"
-          alignItems="center"
-          flex={1}
-        >
-          <Box flexDirection="column">
-            <Box flexDirection="row" alignItems="center">
-              <Box
-                height={10}
-                width={10}
-                borderRadius="m"
-                backgroundColor={
-                  hotspot.status?.online === 'online' ? 'greenOnline' : 'yellow'
-                }
-              />
-              <Text
-                variant="body2Medium"
-                color="offblack"
-                paddingStart="s"
-                ellipsizeMode="tail"
-                numberOfLines={2}
-                maxWidth={210}
-              >
-                {animalName(hotspot.address)}
+      <Pressable onPress={handlePress}>
```

Comment:
Can't we just use a TouchableHighlight?

Reply:
We definitely could, but according to the docs `Pressable` is the way of the future, so wanted to try it out.

![TouchableHighlight · React Native 2021-04-26 at 12 32 34 PM](https://user-images.githubusercontent.com/1760821/116125827-95debe80-a68b-11eb-9d2b-b47daef9b5ca.jpg)

Reply:
Ok haha sounds good